src/main/java/io/zold/api/Network.java

```diff
@@ -45,4 +50,53 @@
      * @return The wallet
      */
     Wallet pull(Long id);
+
+    /**
+     * Simple network implementation.
+     */
+    final class Simple implements Network {
```

Comment:
@paulodamaso what is "simple" about this network?

Reply:
@paulodamaso I think we should name it `RtNetwork` for now and have it in its own class file. It should remain a public class I believe.

Please leave a puzzle for figuring out how to "load" the network. We'll need to load a local JSON file that contains data on all remote nodes that we know about (see [Decisions and Alternatives](https://github.com/zold-io/java-api#decisions-and-alternatives). We must also have a pre-configured set of remote nodes built in (see a footnote somewhere in the whitepaper for that).

Reply:
@llorllale Puzzles added made changes as requested,